.yamato/project-pack.yml

```diff
@@ -14,4 +14,6 @@ pack_{{ project.name }}:
     packages:
       paths:
         - "upm-ci~/packages/**/*"
+  dependencies:
+    - .yamato/project-standards.yml#standards_{{ projects.first.name }}
```

Comment:
What's the reasoning behind this? These two jobs seem completely unrelated.
This PR was slowed down by almost 9 minutes because of this serial execution step. I would have thought we'd instead make this part of the trigger job so it could run in parallel with other jobs.

Reply:
that's `_triggers.yml` right?

Reply:
good shout, done ✅